subjects/digitlen/README.md

```diff
@@ -0,0 +1,46 @@
+## digitlen
+
+### Instructions
+
+Write a function `DigitLen()` that takes two integers as arguments and returns the number of digits in the first `int` by the base of the second `int`.
```

Comment:
Fix: (instructions are not that clear) "Write a function `DigitLen()` that takes two integers as arguments and returns the times the first `int` can be divided by the second until it reaches zero"

Reply:
Thank you José. It is fixed.